src/main/java/uk/gov/hmcts/reform/managecase/service/NotifyService.java

```diff
@@ -67,6 +69,6 @@ private EmailNotificationRequestSuccess sendNotification(EmailNotificationReques
     }
 
     private String createReference() {
-        return "TestReference";
+        return NOTIFY_EMAIL_REFERENCE + System.currentTimeMillis();
```

Comment:
This may generate same references inside application instances running in different pods at the same time.